### PR TITLE
Merge dev xml

### PR DESCRIPTION
**Pull request by joperezr**
cc: @krwq @weshaggard 

Merging dev/xml into dev/api

replaces: #10557

### CODE REVIEW

**Comment by joperezr** [comment]
@dotnet-bot test Innerloop Windows_NT Debug Build and Test please

File: pkg/descriptions.json [hunk 1]
```diff
@@ -1767,6 +1767,21 @@
         ]
     },
     {
+        "Name": "System.Private.Xml",
+        "Description": "Provides provides a fast, non-cached, forward-only way to read and write Extensible Markup Language (XML) data.",
```
Comment (yeiacatl) [inline pkg/descriptions.json]: "Provides provides"

**Comment by joperezr** [comment]
I'm investigating the non-Windows CI failures

**Comment by joperezr** [comment]
@dotnet-bot test Innerloop Windows_NT Debug Build and Test please

File: pkg/descriptions.json (hunk 1 above)
Comment (krwq) [inline pkg/descriptions.json]: good catch, we will fix it as a post PR step

**Comment by joperezr** [comment]
@dotnet-bot test Innerloop OSX Debug Build and Test please

**Comment by joperezr** [comment]
CI was already green, all I did is squash the last few commits. I'm merging now.

**Pull request merged**